docs/_docs/framework/capabilities/optimize/workloads.tobedeleted

```diff
@@ -0,0 +1,123 @@
+  - If you use a service that supports being stopped, but not autostopping, consider using a lightweight flow in [Power Automate](https://learn.microsoft.com/power-automate/getting-started) or [Logic Apps](https://learn.microsoft.com/azure/logic-apps/logic-apps-overview).
+  - If the service can't be stopped, review alternatives to determine if there are any options that can be stopped to stop billing.
+  - Pay close attention to non-compute charges that may continue to be billed when a resource is stopped so you're not surprised. Storage is a common example of a cost that continues to be charged even if a compute resource that was using the storage is no longer running.
+- Does the service support serverless compute?
+  - Serverless compute tiers can reduce costs when not active. Some examples: [Azure SQL Database](https://learn.microsoft.com/azure/azure-sql/database/serverless-tier-overview), [Azure SignalR Service](https://learn.microsoft.com/azure/azure-signalr/concept-service-mode), [Cosmos DB](https://learn.microsoft.com/azure/cosmos-db/serverless), [Synapse Analytics](https://learn.microsoft.com/azure/synapse-analytics/sql/on-demand-workspace-overview), [Azure Databricks](https://learn.microsoft.com/azure/databricks/serverless-compute/).
+- Review service documentation to learn about any cost-related features that could help you optimize your environment or improve cost visibility. Some examples:
+  - Choose [spot VMs](https://learn.microsoft.com/azure/well-architected/cost/optimize-vm#spot-vms) for low priority, interruptible workloads.
+  - Avoid [cross-region data transfer](https://learn.microsoft.com/azure/well-architected/cost/design-regions#traffic-across-billing-zones-and-regions).
+- Does the service support autoscaling?
+  - If the service supports [autoscaling](https://learn.microsoft.com/azure/architecture/best-practices/auto-scaling), configure it to scale based on your application's needs.
+  - Autoscaling can work with autostop behavior for maximum efficiency.
+- Consider automatically stopping and manually starting nonproduction resources during work hours to avoid unnecessary costs.
+  - Avoid automatically starting nonproduction resources that aren't used every day.
+  - If you choose to autostart, be aware of vacations and holidays where resources may get started automatically but not be used.
+  - Consider tagging manually stopped resources. [Save a query in Azure Resource Graph](https://learn.microsoft.com/azure/governance/resource-graph/first-query-portal) or a view in the All resources list and pin it to the Azure portal dashboard to ensure all resources are stopped.
+- Consider architectural models such as containers and serverless to only use resources when they're needed, and to drive maximum efficiency in key services.
+
```

Comment:
Should we highlight to take advantage of the Azure Dev/Test offer? 

•	Leverage the [Azure Dev/Test](https://azure.microsoft.com/en-us/pricing/offers/ms-azr-0148p/) offer that comes with a Visual Studio subscription to take advantage of Azure monthly credits to explore and try various Azure services, benefit from discounted Azure dev/test rates, and enable cost-efficient developing and testing. Although rate optimization strategies can be applied to resources in a development environment, the Azure Dev/Test environment is primarily used for learning and training, development and testing, evaluating proof of concepts, and experimenting and innovating to ensure efficient use of resources.